src/Core/Animation/SkinningData.hpp

```diff
@@ -25,6 +25,9 @@ struct RefData {
     /// Skinning weights.
     Ra::Core::Animation::WeightMatrix m_weights;
 
+    /// The per-bone inverse bind matrices.
+    std::map<uint, Ra::Core::Transform> m_bindMatrices;
```

Comment:
Where here don't you add `inv` in the object  name, as you store inverse matrices ?
This name and its documentation are confusing when you compare it with https://github.com/STORM-IRIT/Radium-Engine/blob/588c6493d49bd5a6322d6f5ed5a185947f8ded57/src/Core/Asset/HandleData.hpp#L33-L34
where here there is no mention of the fact that matrices are inverted.